code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -77,7 +80,10 @@
 
 /mob/living/simple_animal/bot/proc/get_mode()
 	if(client) //Player bots do not have modes, thus the override. Also an easy way for PDA users/AI to know when a bot is a player.
-		return "<b>Sentient</b>"
+		if (!isnull(paicard))

```

Comment:
!isnull returns true if the value is 0, whereas if(paicard) does not. I have no idea why that felt so important at the time, so I'll go ahead and change that.